### PR TITLE
88 python lib and readme update for PIs

### PR DESCRIPTION
**Pull request by GwennanS**
The python package has been distributed and this needs to change to update  everything

README of cc_library and back-end updated with steps to take to run everything on a raspberry pi

https://test.pypi.org/project/scclib/

### CODE REVIEW

File: cc_library/README.md [hunk 1]
```diff
@@ -9,4 +9,26 @@ To make sure all dependencies are loaded, run `pip install -r cc_library/require
 - To check the codestyle, run `flake8 cc_library`.
 
 ### Raspberry Pi Set up
-For setting up a Pi, [a read the `README.md`](src/sciler/README.md) in the sciler package. 
\ No newline at end of file
+For setting up a Pi, [a read the `README.md`](src/sciler/README.md) in the sciler package. 
+
+
+## Set-up Device
+- run `pip install -i https://test.pypi.org/simple/ scclib`
```
Comment (IssaHanou) [inline cc_library/README.md]: watch the space

File: cc_library/src/sciler/README.md [hunk 2]
```diff
@@ -9,23 +9,14 @@ This mainly consists of:
 - `device_manual.md` the manual for writing configuration files for devices
 - `LICENSE.md` the license with which this library complies
 
-## Set-up device with package.
-- create custom device script, which should inherit from `Device`, add it to `cc_library/src/scripts`
-- write configuration for the the device, according to `device_manual.md`, add it to `cc_library/src/scripts`
-- test the device script, by altering `device_test.py`, import the class on line 7, change the device class in line 15 and the config file name in line 21
+## Set-up Device
+- run `pip install -i https://test.pypi.org/simple/ scclib`
+- create custom device script, which should inherit from Device
```
Comment (IssaHanou) [inline cc_library/src/sciler/README.md]: create a custom device script, with a class inheriting from the Device superclass, whose main method is called in the script

File: cc_library/README.md (hunk 1 above)
Comment (GwennanS) [reply cc_library/README.md]: from `https://test.pypi.org/simple/` install `scclib` is correct

File: cc_library/py-scc/README.md [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
-
+                                                         
 To make sure all dependencies are loaded, run `pip install -r cc_library/requirements.txt`.
 
 ### Checks
 - To check unittests, run `python -m unittest discover`.
-- To update formatting, run `black py-scc`.
-- To check the codestyle, run `flake8 py-scc`.
+- To update formatting, run `black cc_library`.
+- To check the codestyle, run `flake8 cc_library`.
```
Comment (mrlomar) [inline cc_library/py-scc/README.md]: shouldn't this remain py-scc?

File: cc_library/py-scc/src/sciler/scclib/app.py [hunk 4]
```diff
@@ -26,7 +26,7 @@ def __init__(self, config, device):
             os.mkdir("logs")
         filename = "logs/log-" + datetime.now().strftime("%d-%m-%YT--%H-%M-%S") + ".txt"
         logging.basicConfig(
-            level=logging.INFO,
+            level=logging.FATAL,
```
Comment (mrlomar) [inline cc_library/py-scc/src/sciler/scclib/app.py]: Maybe change this back since the written logs will also not record everything anymore.

**Pull request merged**